src/ui-lib/sass/04-objects/_container-banner.scss

```diff
@@ -0,0 +1,4 @@
+.grav-o-container-banner {
+  padding: $grav-sp-xl $grav-sp-m $grav-sp-m;
```

Comment:
perhaps you want to create a new `$grav-sp-inset-squished-xl` and use that one?

Reply:
You also want to showcase this class next to the `organism-section` and write some documentation for it, this seems to be describing an alternative type of container compared to `grav-o-container` so you might want to make sure this is well understood.

Reply:
inset-squished is 2 dimensional, so its top/bottom & right/left. And what I'm doing is setting top as XL, right/left as M, and bottom as M..so I don't think adding a new one is necessary here? Correct me if I'm wrong.

Reply:
no but what I'm suggesting is to have more or less the same padding that the other container is using, that is using same vertical/horizontal padding (maybe xl/l ?)

Reply:
In that way you also will help the spacing for the icon list (see my other comment)